docs/framework/wcf/diagnostics/exceptions-reference/identitymodel-exceptions.md

```diff
@@ -239,7 +239,7 @@ This topic lists all exceptions generated by IdentityModel.
 |TraceCodeSecurityPendingServerSessionRemoved|A pending security session was made active by the server.|  
 |TraceCodeImportSecurityChannelBindingExit|Finished Security ImportChannelBinding.|  
 |X509CertStoreLocationNotValid|The StoreLocation must be either LocalMachine or CurrentUser.|  
-|SettingdMayBeModifiedOnlyWhenTheWriterIsInStartState|The writer settings may be modified only when the writer is in the Start state.|  
+|SettingsMayBeModifiedOnlyWhenTheWriterIsInStartState|The writer settings may be modified only when the writer is in the Start state.|  
```

Comment:
This one might be a product bug, but I didn't try to trace down to the code

Reply:
ping @HongGit for this change. "Settings" vs. "Settingd" in the resource string.

I can't find any instance of either spelling in our docs.

The rest of the PR looks fine, but I'm not sure which is correct here.

Reply:
I don't think the resource name is that important anyway, so can merge as-is.